### ISSUE TITLE
BE_SMTPSend crashes script engine with FileMaker Server 2023 on Ubuntu 22.04.2

### ISSUE THREAD

**Issue by skgreg** [GoyaPtyLtd/BaseElements-Plugin#216]
We're using version 4.2.6 on two different servers that were recently upgraded to FileMaker Server 2023 (20.1.2) and Ubuntu 22.04.2. Scripts called with Perform Script on Server that were previously working are now causing the script engine to crash when we call the BE_SMTPSend function. We've tested with the minimum required parameters and verified the same script works successfully when run from FileMaker Pro.

**Comment by nickorr** [comment]
Greg,

We've seen this as well, and can confirm it.  I don't have a fix for it yet, but will let you know when we do.  Our workaround was to use native SMTP instead for now.

Cheers,
Nick

**Comment by petrowsky** [comment]
Are there larger issues with fms 20.1.2.207?

I'm running a local dev server within a Parallels VM within **ARM** and using 22.04.2. Is the plugin not running on ARM? Only AMD?
```
Error   474     fms.local       Plug-in failed to load: BaseElements.fmx
```

**Comment by nickorr** [comment]
Yes, current build is x86 only for linux, arm support will come soon.

**Comment by vlekkenconsult** [comment]
Hi, when is arm support on linux ubuntu expected?

**Comment by nickorr** [comment]
For the SMTP issue, this is not a final build, but some testing ( not in production ) would be helpful :

https://goya.com.au/files/beplugin/5.0.0/BaseElements.fmx.zip

**Comment by vlekkenconsult** [comment]
Any progress on arm support? Eager to test it if you have already something available for that :-)

**Comment by nickorr** [comment]
Can you test out the new version 5.1.0b3 ( 5.1.0.6 ) and report any issues here : 

https://github.com/GoyaPtyLtd/BaseElements-Plugin/blob/main/docs/Downloads.md
